src/Core/JavaArchive.php

```diff
@@ -110,6 +110,9 @@ public function __construct(string $jarFile, array $options = [])
             if (empty($attribute)) {
                 continue;
             }
+            if (!strpos($attribute, ':')) {
```

Comment:
The strpos function will return 0 or `false`. This expression is true both. 
Please change  `!strpos(...)` to `strpos(...) !== false`.